bot/resources/tags/xy-problem.md

```diff
@@ -1,7 +1,7 @@
 **xy-problem**
 
-Asking about your attempted solution rather than your actual problem.
+The XY problem can be summarised as asking about your attempted solution, rather than your actual problem.
 
 Often programmers will get distracted with a potential solution they've come up with, and will try asking for help getting it to work. However, it's possible this solution either wouldn't work as they expect, or there's a much better solution instead.
 
-For more information and examples: http://xyproblem.info/
+For more information and examples, see http://xyproblem.info/
```

Comment:
```suggestion
For more information and examples see http://xyproblem.info/
```

Reply:
Same as above. I don't see how this comma is wrong.